decidim-core/app/packs/src/decidim/sticky_header.js

```diff
@@ -0,0 +1,17 @@
+let prevScroll = window.scrollY;
+const stickyHeader = document.getElementById("sticky-header");
+
+document.addEventListener("scroll", () => {
+  // if a subelement is not visible it has no offsetParent
+  const header = document.getElementById("main-bar").offsetParent;
+  if (header) {
+    let currentScroll = window.scrollY;
+    if (prevScroll > currentScroll || currentScroll < stickyHeader.offsetHeight) {
+      stickyHeader.style.top = 0;
+    } else {
+      stickyHeader.style.top = `-${stickyHeader.offsetHeight}px`;
+      console.log(header.offsetHeight);
+    }
+    prevScroll = currentScroll;
+  }
+});
```

Comment:
```suggestion
if(stickyHeader) {
  document.addEventListener("scroll", () => {
    // if a subelement is not visible it has no offsetParent
    const header = document.getElementById("main-bar").offsetParent;
    if (header) {
      let currentScroll = window.scrollY;
      if (prevScroll > currentScroll || currentScroll < stickyHeader.offsetHeight) {
        stickyHeader.style.top = 0;
      } else {
        stickyHeader.style.top = `-${stickyHeader.offsetHeight}px`;
      }
      prevScroll = currentScroll;
    }
  });
```

Let's add guard clause in case the sticky header is not present and also remove the log

Reply:
Done